.github/workflows/scrape.yml

```
@@ -214,6 +214,7 @@ jobs:
   scrape-hass-pathways:
     name: Scrape HASS pathways
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

When it works, it usually takes well under 1 min. I think we should set this as 1min instead
```suggestion
    timeout-minutes: 1
```